backend/coins/btc/transaction_test.go

```diff
@@ -44,6 +46,65 @@ func mustKeypath(t *testing.T, keypath string) signing.AbsoluteKeypath {
 	return kp
 }
 
+type MockTransaction struct {
```

Comment:
Consider using `moq` to auto-generate a mock struct, like

https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/b9cc719ea68ffffa3ed961d56504b2111dc4dd05/backend/accounts/account.go#L67

which generates backend/accounts/account/mocks/account.go.

Then you can can use it locally in the test below and set the functions directly, so it's closer to the test, like 

```
acccount.transactions = mocks.InterfaceMock{
    SpendableOutputsFunc: func() (...) { ... },
    ....
}
```

Reply:
Yep good idea, done